VespaGAS/Models/Gasoline.swift

```diff
@@ -1 +1,11 @@
-typealias Gasoline = Volume
+import Foundation
+
+struct Gasoline: Volume {
+    var amount: Float
```

Comment:
Apparently I can't use let. Compiler complains about these structs not conforming anymore to the Volume protocol